### DISCUSSION TITLE
Any events available to monitor Clipboard usage?

### DISCUSSION THREAD

**Discussion by CalvinAllen** [VsixCommunity/Community.VisualStudio.Toolkit#359] [Q&A] [answered]
Is there anything to monitor if a user copies something to the clipboard? Any events to subscribe to or anything?

**Comment by reduckted** [comment, marked as answer]
If you're willing to handle the copy yourself, there are a couple of options. 
* You could intercept the "copy" command: https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/blob/9c32305c83827d0bfd7c85777f9e79fc71e22c2b/src/toolkit/Community.VisualStudio.Toolkit.Shared/Commands/Commands.cs#L96
* You could add a command handler for the "copy" command and do the copying yourself: https://github.com/madskristensen/Tweakster/blob/master/src/TweaksterShared/Tweaks/Editor/CopyWithoutIndentation.cs

**Reply by CalvinAllen** [reply]
Yeah that may work just fine, thanks!
